lore/admins/reboot-zuul.md

```diff
@@ -0,0 +1,82 @@
+---
+name: rebooting zuul
+layout: default
+---
```

Comment:
Don't forget to add a permalink:

```markdown
permalink: /lore/admins/reboot-zuul/
```

**NOTE:**
You will also need to link here from lore/admins/README.md